### DISCUSSION TITLE
Access admin while using custom strategy

### DISCUSSION THREAD

**Discussion by elmage** [payloadcms/payload#1434] [General]
I'm working on user authentication using a custom strategy I wrote.
Everything is fine except, the user still can't access the admin dashboard.
A request to `/api/access` shows that the user has all access to all collection but `canAccessAdmin` is set to `false`.
Do custom strategies not allow for users to access admin?

**Comment by jmikrut** [comment]
Hey @elmage — you can 100% set up a custom auth strategy to be able to access the Payload admin UI. You may just not have your strategy configured right to correctly identify your user. These are quite complex to set up on your own, but I know there are a few plugins out there that you could look at. Try searching around here and on Discord to see what you can find!

**Reply by elmage** [reply]
Thanks for the info, @jmikrut.
I'll head over to Discord.
I've just been at this issue for too long.
